src/api/json/catalog.json

```diff
@@ -2756,6 +2756,12 @@
       "fileMatch": ["mycode.json"],
       "url": "https://json.schemastore.org/mycode.json"
     },
+    {
+      "name": "MyPy",
+      "description": "MyPy, a Python type checker",
```

Comment:
hate to nitpick/drive-by, but should this be "Mypy" instead? That seems to be the casing on the website: https://mypy.readthedocs.io/en/stable/.

Reply:
I think it's actually mypy, with Mypy only being at the start of a sentence.

Reply:
```suggestion
      "name": "mypy",
      "description": "mypy, a Python type checker",
```